core/src/main/java/org/frankframework/scheduler/job/CheckReloadJob.java

```diff
@@ -33,6 +33,9 @@
 import org.frankframework.util.MessageKeeper.MessageKeeperLevel;
 import org.frankframework.util.SpringUtils;
 
+/**
+ * Frank!Framework job which periodically looks in the {@code IBISCONFIG} table to see if a new {@link Configuration} should be loaded.
```

Comment:
`{@code ...}`, of `{@literal ...}`? 🤔 
Het is mij nog niet helemaal duidelijk wanneer we welke het beste kunnen gebruiken en wat het beste werkt met de Frank!Doc.